### PR TITLE
[Remove Spring extender usages] LPS-159184

### PR DESCRIPTION
**Pull request by yuhai**
@dantewang

### CODE REVIEW

File: modules/apps/commerce/commerce-product-type-virtual-service/src/main/java/com/liferay/commerce/product/type/virtual/service/impl/CPDefinitionVirtualSettingServiceImpl.java [hunk 1]
```diff
@@ -201,14 +211,13 @@ private void _checkCommerceCatalog(long cpDefinitionId, String actionId)
...
 	}
 
-	private static volatile ModelResourcePermission<CommerceCatalog>
-		_commerceCatalogModelResourcePermission =
-			ModelResourcePermissionFactory.getInstance(
-				CPDefinitionVirtualSettingServiceImpl.class,
-				"_commerceCatalogModelResourcePermission",
-				CommerceCatalog.class);
+	@Reference(
+		target = "(model.class.name=com.liferay.commerce.product.model.CommerceCatalog)"
```
Comment (dantewang) [inline modules/apps/commerce/commerce-product-type-virtual-service/src/main/java/com/liferay/commerce/product/type/virtual/service/impl/CPDefinitionVirtualSettingServiceImpl.java:215]: @yuhai My only concern is why this class uses the resource permission of `CommerceCatalog`, but for the conversion it's good to go.

**Review by dantewang — APPROVED** [review]

**Comment by tinatian** [comment]
@yuhai Code looks good to me, wait for QA to scan test results.

Tina

**Comment by tinatian** [comment]
@catarineaguiar Please help me scan test results.

Thanks.
Tina

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

**Comment by shuyangzhou** [comment]
ci:forward

**Pull request closed without merging**